src/Workspaces/Core/Portable/Options/EditorConfig/EditorConfigFileGenerator.cs

```diff
@@ -90,119 +86,5 @@ string GetEditorConfigString(IOption option, IEditorConfigStorageLocation2 edito
                 return editorConfigString;
             }
         }
-
-        public static void AppendNamingStylePreferencesToEditorConfig(NamingStylePreferences namingStylePreferences, string language, StringBuilder editorconfig)
```

Comment:
The deleted code here is just moved to a separate partial declaration down in the shared layer. Unfortunately, I missed including that new file in this commit - it is in one of the following commits, without any semantic changes.